src/test/java/io/redisearch/client/ClientTest.java

```diff
@@ -1137,7 +1137,12 @@ public void testConfig() throws Exception {
         cl.setConfig(ConfigOption.ON_TIMEOUT, "fail");
         assertEquals("fail", cl.getConfig(ConfigOption.ON_TIMEOUT));
 
-        assertFalse(cl.setConfig(ConfigOption.ON_TIMEOUT, "null"));
+        try {
```

Comment:
@ashtul can we avoid backward?

Reply:
@gkorland yes, we can keep on returning a simple string.
The reason for the fix was a leak of the error string. We can change back to it while freeing the string properly.